### PR TITLE
Adding Full Stack Development Heading + book

### PR DESCRIPTION
**Pull request by Recelis**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [ x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/book-recommendations/index.md [hunk 1]
```diff
@@ -128,6 +128,11 @@ title: Books to Read for Programmers
 - [Amazon Smile](https://www.amazon.com/DevOps-Handbook-World-Class-Reliability-Organizations/dp/1942788002/ref=sr_1_4?s=books&ie=UTF8&qid=1539733859&sr=1-4&keywords=devops+handbook)
 - ISBN-13: 978-1942788003
 
+## Full Stack Development
+* Pro Mern Stack: Full Stack Web App Development with Mongo, Express, React, and Node*
```
Comment (amandeepmittal) [inline guide/english/book-recommendations/index.md]: Please remove the asterisk.

File: guide/english/book-recommendations/index.md [hunk 2]
```diff
@@ -128,6 +128,11 @@ title: Books to Read for Programmers
...
+- https://www.amazon.com/Pro-MERN-Stack-Development-Express/dp/1484226526
```
Comment (amandeepmittal) [inline guide/english/book-recommendations/index.md]: Add the link in correct format. Hint: see how other links are added in this document.

**Review by amandeepmittal — CHANGES_REQUESTED** [review]

**Review by paulywill — COMMENTED** [review]
@amandeepmittal Is correct! Please fix these up.

**Review by amandeepmittal — APPROVED** [review]
LGTM now!

**Pull request merged**